lib/acpi/Kconfig

```diff
@@ -49,3 +43,15 @@ config ACPI_HID_LEN_MAX
 	default 12
 	help
 	  Size of HID string.
+
+config ACPI_IRQ_VECTOR_MAX
```

Comment:
All ACPI Kconfig symbols should be inside the `if ACPI` block. This includes also the existing HID, however in that case I think you can remove the Kconfig option completely since the ACPI spec defines a maximum worst-case length for this. That said, I also don't see you populating or using the hid value anywhere, so you should probably just remove it until there's code that needs it.